amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/RetrievalConfig.java

```diff
@@ -121,6 +121,13 @@ public RetrievalConfig initialPositionInStreamExtended(InitialPositionInStreamEx
         return this;
     }
 
+    public RetrievalConfig retrievalSpecificConfig(RetrievalSpecificConfig retrievalSpecificConfig) {
```

Comment:
Does this get called directly by the customer when they initialize their config? Don't see it used in this PR

Reply:
Yes this will be set by customer - PollingConfig and FanoutConfig